motioneye/config.py

```diff
@@ -1695,11 +1699,13 @@ def motion_camera_dict_to_ui(data):
         elif ' webhook ' in e:
             e = shlex.split(e)
 
-            if len(e) < 3:
+            if len(e) < 5:
```

Comment:
Does this mean, that even though "accept" and "user agent" headers were not made required (at least the tooltip suggests that), old webhook configs which have not those defined are ignored completely and the user is required to redo webhook configs? If the added params were left as the last ones (i.e. moving them after "url" param), would it then be possible to just set "accept" and "user agent empty" if they do not occur in the config?

Reply:
I think this is not an issue. The event string with the now 5 fields is constructed on startup as of `on_event_*.append` above, so it will have the 5 fields, even that two of them will be empty fields at first.

Reply:
This entire section will need adjustment if we make Accept and User-Agent headers optional (right now they would be required, which I agree can break existing users and should be adjusted to not impact them)

Reply:
Ah right, the code to generate the event strings is generated only when settings are saved in the GUI, the strings are stored in the camera configs. Just tested it.

If this is so, then I suggest:
```py
            if len(e) < 3:
                continue

            ui['web_hook_notifications_enabled'] = True
            ui['web_hook_notifications_url'] = e[-1]

            # support pre-v0.43.0 configs
            if len(e) < 5:
                ui['web_hook_notifications_http_method'] = e[-2]
                continue

            ui['web_hook_notifications_http_method'] = e[-4]
            ui['web_hook_notifications_accept'] = e[-3]
            ui['web_hook_notifications_user_agent'] = e[-2]
```
Probably it's also better to add the new parts to the beginning of the string, so that reverting to older motionEye is still possible (since indices are used from the back of the string here).

Reply:
Yeah I just tried doing it from the front but e[1] turns out to not be of the webhook notification params... it's the very start of the on_event_start string. In this case your suggestion makes sense, i.e. continue indexing from the back but make it a convention to introduce new fields in the future to the "beginning" (after webhook). I'll push a new commit accordingly

Reply:
Updated the PR